app/controllers/comments_controller.rb

```diff
@@ -221,7 +221,12 @@ def hide
     authorize @comment
     @comment.hidden_by_commentable_user = true
     @comment&.commentable&.update_column(:any_comments_hidden, true)
-    if @comment.save
+
+    # Hide notification related to that comment
+    @comment_notification = Notification.find_by(notifiable_type: "Comment", user_id: current_user.id, notifiable_id: params[:comment_id])
```

Comment:
I could not find existing test for this hide action or even the controller itself so I might need little help on that.